### DISCUSSION TITLE
Exporting edited mods?

### DISCUSSION THREAD

**Discussion by kiraira** [xivdev/Penumbra#121] [Q&A] [answered]
Created a mod in TexTools and tweaked it in Penumbra, and I want to share it with some friends. Is there any way to export it as a .ttmp file? If not, what is the way that mods are created/exported with Penumbra? Apologies if this is a dumb question or has been answered already.

**Comment by Ottermandias** [comment, marked as answer]
Penumbra just has a folder in your mod root folder. That's just a filesystem thing. You can copy folders and files from there, separately or grouped, you can zip them up and send them, and the other person only has to put that folder in his mod root folder.

**Reply by kiraira** [reply]
Thank you!
